<h1 align='center'>
  ‚ùó An√°lise dos dados da Covid-19 pelo mundo
  
  [![linkedin](https://img.shields.io/badge/author-victorbarbosa-blue.svg)](https://www.linkedin.com/in/victorxbarbosa/) 
  [![](https://img.shields.io/badge/python-3.7+-yellow.svg)](https://www.python.org/downloads/release/python-365/) 
  [![GPLv3 license](https://img.shields.io/badge/License-GPLv3-red.svg)](http://perso.crans.org/besson/LICENSE.html)
  [![contributions welcome](https://img.shields.io/badge/contributions-welcome-brightgreen.svg?style=flat)](https://github.com/victorxbarbosa)
  
</h1>

<p align='center'>

  <img src="https://user-images.githubusercontent.com/100004569/189538175-e314f66c-cf79-49fc-8eaa-c7a16d488dc3.png">

</p>

## üöÄ Tecnologias
Este projeto utiliza as seguintes tecnologias:

- [Python](https://www.python.org/)
- [Pandas](https://pandas.pydata.org/)
- [Numpy](https://numpy.org/)
- [Seaborn](https://seaborn.pydata.org/)
- [Matplotlib](https://matplotlib.org/)
- [Google Colab](https://research.google.com/colaboratory/)

## üíªProjeto
Em 31 de dezembro de 2019, a Organiza√ß√£o Mundial da Sa√∫de (OMS) foi alertada sobre v√°rios casos de pneumonia na cidade de Wuhan, prov√≠ncia de Hubei, na Rep√∫blica Popular da China. Tratava-se de uma nova cepa (tipo) de coronav√≠rus que n√£o havia sido identificada antes em seres humanos.

Uma semana depois, em 7 de janeiro de 2020, as autoridades chinesas confirmaram que haviam identificado um novo tipo de coronav√≠rus. Os coronav√≠rus est√£o por toda parte. Eles s√£o a segunda principal causa de resfriado comum (ap√≥s rinov√≠rus) e, at√© as √∫ltimas d√©cadas, raramente causavam doen√ßas mais graves em humanos do que o resfriado comum.

Ao todo, sete coronav√≠rus humanos (HCoVs) j√° foram identificados: HCoV-229E, HCoV-OC43, HCoV-NL63, HCoV-HKU1, SARS-COV (que causa s√≠ndrome respirat√≥ria aguda grave), MERS-COV (que causa s√≠ndrome respirat√≥ria do Oriente M√©dio) e o, mais recente, novo coronav√≠rus (que no in√≠cio foi temporariamente nomeado 2019-nCoV e, em 11 de fevereiro de 2020, recebeu o nome de SARS-CoV-2). Esse novo coronav√≠rus √© respons√°vel por causar a doen√ßa COVID-19.

Portanto, Este √© um dos projetos do curso Data Science na Pr√°tica da [Sigmoidal](https://sigmoidal.ai/) com o objetivo de fazer a coleta dos dados, a limpeza dos dados, realizar uma an√°lise explorat√≥ria tanto continental quanto aos pa√≠ses mais afetados, para entender como se desencadeou esta doen√ßa que parou o mundo.
- [Dados do projeto](https://ourworldindata.org/)
- [Notebook do projeto](https://github.com/victorxbarbosa/Analise-da-Covid19-no-mundo/blob/main/Estudo_de_caso_da_COVID_19.ipynb)
